config/_default/params.en.yaml

```diff
@@ -3,7 +3,7 @@ meta_title: Getting Started with Datadog
 meta_description: Datadog, the leading service for cloud-scale monitoring.
 disclaimer: "This page is not yet available in English, we are working on its translation. <br> May you have any questions or feedback about our current translation project, <a href=\"https://docs.datadoghq.com/fr/help/\">feel free to reach out to us!</a>"
 announcement_banner:
-  desktop_message: "Read the 2024 State of Cloud Costs Study!"
-  mobile_message: "Read the State of Cloud Costs Study!"
-  external_link: "https://www.datadoghq.com/state-of-cloud-costs/?utm_source=inbound&utm_medium=corpsite-display&utm_campaign=dg-cloudcostmgmt-ww-research-banner-report-state-cloud-costs"
+  desktop_message: "Datadog Webinar: Always-On Code Profiling"
```

Comment:
Just want to make sure that this title is correct, as the webinar itself has a significantly different title